### ISSUE TITLE
ReferenceError with generator

### ISSUE THREAD

**Issue by yaru22** [babel/babel#1395]
I'm using Babel 5.1.13.

If I compile the following ES6 code, 
```js
import Promise from 'bluebird';
import r from 'rethinkdb';

Promise.coroutine(function *() {
  let allMembers = [1,2,3];
  let query = r.table('User').getAll(...allMembers);
})();
```

I get the following code:
```js
'use strict';

var _interopRequireDefault = function (obj) { return obj && obj.__esModule ? obj : { 'default': obj }; };

var _Promise = require('bluebird');

var _Promise2 = _interopRequireDefault(_Promise);

var _r = require('rethinkdb');

var _r2 = _interopRequireDefault(_r);

_Promise2['default'].coroutine(regeneratorRuntime.mark(function callee$0$0() {
  var allMembers, query;
  return regeneratorRuntime.wrap(function callee$0$0$(context$1$0) {
    while (1) switch (context$1$0.prev = context$1$0.next) {
      case 0:
        allMembers = [1, 2, 3];
        query = (_r$table = _r2['default'].table('User')).getAll.apply(_r$table, allMembers);

      case 2:
      case 'end':
        return context$1$0.stop();
    }
  }, callee$0$0, this);
}))();
```

In the generated code, it's using `_r$table` without declaring it. Thus, it results in ReferenceError in runtime. So it looks like a bug.

Is there a workaround for this?

**Comment by sebmck** [comment]
Upgrade to latest.

**Comment by sebmck** [comment]
Actually nevermind, I haven't pushed a new release in a while and the fix isn't available yet. I'll close this once I push out a new release which should be in the next couple hours.

**Comment by yaru22** [comment]
Thanks for super quick response @sebmck. Look forward to the fix.

**Comment by sebmck** [comment]
Fixed as of 5.2.0. Thanks!
